radixdlt-core/radixdlt/src/main/java/org/radix/network/messaging/Message.java

```diff
@@ -96,7 +94,7 @@ protected Message() {
 		this(Time.currentTimestamp());
 	}
 
-	protected Message(long timestamp) {
+	private Message(long timestamp) {

```

Comment:
Why is this private? Additionally, use of `Time.currentTimestamp()` will probably make this class untestable.